### ISSUE TITLE
TODOs from Fri 4/3 meeting

### ISSUE THREAD

**Issue by michaelchughes** [tufts-ml/covid19-forecasting#3]
## Input from CHIME/IHME models of presenting patients (@panlybero )

* Specify as input files:  "results/chime_output_1.csv" (columns: timestep, n_Presenting)
* Can use random seeds for this too:
* * chime_1.csv
* * chime_2.csv

## Running on Tufts HPC cluster ( @gvisani )
* [x] Given a concrete set of params (json file), we want to be able to kickoff 1000 trials with specified random seeds, save those to a output directory with specified name

## Reporting results (@panlybero )

* [x] Write script that consumes folder full of results files, and produces summary statistics at each day for each stage
---- INPUT: results-1.csv, results-2.csv, .... results-101.csv  (columns: n_InICU, n_OnVentInICU, ...)
---- OUTPUT: summary.csv (columns: n_InICU_2.5, n_InICU_5, n_InICU_50) [explode each colung in INPUT to one for each percentile)

## Visualization (@diana)

* [x] Write script that converts summary.csv of a simulation to set of plots
* [ ] Visualization to explore how parameters matter (slider where adjust mean time ICU for recovery)
- [x] Update aesthetics (pretty-print strings, update dimensions of plots, layout)
- [x] Add checkboxes to Dash app to show/hide plots based on name
- [x] Add series for actual data to compare with predictions/forecast
- [x] Add column for actual date information rather than generic time step
- [x] Allow for ability to update visualization based upon parameter change (proof of concept: recovery probability)

**Comment by michaelchughes** [comment]
## Updated proposal for summaries 

For each summary statistic of interest (e.g. mean, stddev, percentile=050.00, percentile=001.00, etc), we save a SEPARATE CSV file named "summary-{summary_name}.csv"


See #7 for a PR that does this simply (PR 7 has been merged into master now)

Example summary-{}.csv files:

https://github.com/tufts-ml/covid19-forecasting/blob/4b5758fb6c39c1e3caed4c08c8f6ee158b8c1e86/example_output/summary-percentile%3D002.50.csv

https://github.com/tufts-ml/covid19-forecasting/blob/4b5758fb6c39c1e3caed4c08c8f6ee158b8c1e86/example_output/summary-percentile%3D097.50.csv

## Updated proposal for Diana's visualizaton code

Then, our downstream plotting code can be written like this: 

```
python plot_forecasts \
    --summary_file summary-mean.csv \
    --summary_upper_file summary-percentile=097.50.csv \
    --summary_lower_file summary-percentile=002.50.csv \
    --columns n_InICU,n_OnVentInICU
```

**Comment by michaelchughes** [comment]
Merged #8 to address first of visualization todos. We have working plotly plots!

**Comment by michaelchughes** [comment]
Current version addresses these issues. Thanks all!
